### ISSUE TITLE
[Bug] Freezes when newline is append to Translation Key

### ISSUE THREAD

**Issue by loganisitt** [FlineDev/RemafoX#65]
## Steps to Reproduce:

I accidentally added a new line to the translation key on several occasions. This freezes the file and forces me to restart Xcode.

1. Add Translation
1. Enter the Translation Key followed by tapping Return

## Expected Behavior:

Corrects or warns me about invalid input.

## Current Behavior:

Freezes the file.

## Environment

<details>
<summary>Show environment details</summary>

- App Version: 1.1.1 (Build 55)
- System Version: 13.0.1 (Build 22A400)
- System CPU: Apple M1 Max
- Tier: Pro

</details>

**Comment by Jeehut** [comment]
@loganisitt Thank you for reporting this and sorry for the late response.

I just tested this out and I can't reproduce said issue. When I enter the text "Main" then press the return key and then "Title" into the "Translation Key" field, after finishing up the Add Translation form and pressing "Insert as String" everything works in my case. May I ask what version of Xcode you have experienced this in?

As per the suggested solution (independent from being able to reproduce the issue): I think it makes sense to at least warn users within the Add Tranlation form when they are using a newline character in it. I will add such a warning. I'll keep you updated once I implemented it.

**Comment by Jeehut** [comment]
@loganisitt As suggested in your original request, I've added a visual warning when a newline is used in the key:
<img width="1052" alt="Bildschirm­foto 2022-12-05 um 14 10 43" src="https://user-images.githubusercontent.com/6942160/205554377-afdc3526-3ba8-4868-bd62-0803c0c38749.png">

I still can't reproduce the freezing file issue, but if you don't use a newline in the key anymore thanks to this, it also shouldn't be an issue anymore. I'd still be happy to fix that as well in case I can do anything there (might also be an Xcode bug?), I just need to be able to reproduce it, so feel free to provide more details.

Closing this (for now), the fix will be included in `1.2.0` to be released soon.
